components/instructions/ExecuteAllInstructionButton.tsx

```diff
@@ -0,0 +1,147 @@
+import { useEffect, useState } from 'react'
+import {
+  InstructionExecutionStatus,
+  Proposal,
+  ProposalTransaction,
+  ProposalState,
+} from '@solana/spl-governance'
+import React from 'react'
+import { CheckCircleIcon, PlayIcon, RefreshIcon } from '@heroicons/react/solid'
```

Comment:
group dependencies imports at the top, local imports below for better visibility